core/src/main/java/cz/xtf/core/helm/HelmClients.java

```diff
@@ -0,0 +1,20 @@
+package cz.xtf.core.helm;
+
+import cz.xtf.core.config.OpenShiftConfig;
+
+public class HelmClients {
+
+    public static HelmBinary adminBinary() {
+        return getBinary(OpenShiftConfig.adminToken(), OpenShiftConfig.adminUsername(), OpenShiftConfig.namespace());
```

Comment:
these methods should probably delegate to instantiable HelmBinaryManager

HelmBinaryManagerFactory.INSTANCE.getHelmBinaryManager().adminBinary()